htdocs/editor_tab.js

```diff
@@ -1412,9 +1455,24 @@ var editor = function () {
                         $(".history i",$(node.element)).addClass("button-disabled");
                     }
                     $tree_.tree('openNode', node);
+
+                    return Promise.resolve();
```

Comment:
This is returning a promise of undefined instead of just returning undefined, so it doesn't really do anything.